lib/exq.ex

```diff
@@ -25,14 +26,14 @@ defmodule Exq do
   end
 
   def enqueue(pid, queue, worker, args) do
-    GenServer.call(pid, {:enqueue, queue, worker, args})
+    GenServer.call(pid, {:enqueue, queue, worker, args}, Config.get(:redis_timeout, 5000))
```

Comment:
So this is setting the timeout for the generic GenServer call?  Just curious, wouldn't the Redis connection be timed in the Enqueuer?